logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/ConversationEventReceiver.kt

```diff
@@ -135,6 +139,7 @@ class ConversationEventReceiverImpl(
 
                     else -> kaliumLogger.withFeatureId(EVENT_RECEIVER).e("$TAG - Failure when processing message: $it")
                 }
+                handleFailedProteusDecryptedMessage(event)
```

Comment:
Shouldn't this be called only when `is ProteusFailure ->`? Right now it's called for every failure type 🤔

Reply:
After private discussion with Jacob, we decided to group all these errors into the same generic Decryption Failure message. Also the current designs for this error handling are quite vague and we are not aiming to inform the final user about all the nuances of our own internal error handling.